docs/tensorboard_in_notebooks.ipynb

```diff
@@ -4,7 +4,6 @@
   "metadata": {
```

Comment:
<p>Can you please commit the notebook from the clean state? Most users won't be seeing this message.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/2759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Yes, I made a new PR with the clean state of the notebook. I'm sorry that I had to delete this PR since there were to many commits and I didn't want to make multiple merges for a minor change. The new PR is here: <a href="https://github.com/tensorflow/tensorboard/pull/2803" target="_blank">https://github.com/tensorflow/tensorboard/pull/2803</a>. Please, look into that. </p><p> </p><p>Also, sorry for not being responsive. </p>